### DISCUSSION TITLE
Customize the Tab component template

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#10741] [General - Components / Directives / etc] [answered]
Is it possible to customize the visuals of a Tab using template or something along those line? I didn't see any example in the docs so I figured it was not supported. I'd like to customize the Tab's visuals to simply make the icon us a 'small icon' rather than it's current default. 

Ideally i'd like the icons to be above the text but I'm fine if for whatever reason the icon has to be next to the text. Which i would also like to make the fond smaller as well for the tabs. Any help is appreciated.

![image](https://user-images.githubusercontent.com/1416749/133937246-c63359ee-84ce-40ae-9e14-07b2f2c9e001.png)

**Comment by ontwikkelfabriek** [comment]
> Is it possible to customize the visuals of a Tab

Yes it is very possible, but requires CSS/flexbox skills. Quasar uses the **BEX** methodology to style it's components. Also inside it's components **flexbox** is used.  So with the help of the chrome inspector you can style your tab components **whatever** you want.

> Ideally i'd like the icons to be above the text 

Did you look at the Qtab examples, because there's a demo that has a qtab with the text above the icon:
https://quasar.dev/vue-components/tabs#individual-colors

**Comment by exninja** [comment, marked as answer]
You can do this without css by nesting icon and text within the q-tab element.

Default:
```
          <q-tab name="tab1" icon="myicon" label="mylabel" />
```

Override
```
          <q-tab name="tab1">
            <q-icon
              name="myicon"
              size="xs"
            />
            <span class="text-h3">mylabel</span>
          </q-tab>
```

**Reply by JokerMartini** [reply]
Awesome thank you
